source/b_usermanagement.rst

```diff
@@ -0,0 +1,68 @@
+.. _OrganisationsAnchor:
+
+=============
+Organisations
+=============
+
+All data in Lizard is linked to an organisation.
+Organisations are the backbone of our authorisation model.
+There are three types of authorisation options that can be applied to your data: 
+
+* **Public**: everyone can see your data 
+* **Common**: everyone with login credentials to Lizard can see your data 
+* **Private**: everyone with login credentials to Lizard AND user rights to your organisation can see your data
+
+.. image:: /images/b_usermanagement_01.png
+
+=====
+Roles
+=====
+
+We have 4 roles and 3 different types of privileges. 
+
+1. A **user**, who can only *read* data
+2. A **supplier**, who can *read* data and change (*'write'*) his or her own data
+3. An **administator**, who can *read* data and change (*'write'*) all organisation's data. 
+4. A **manager**, who can *manage* other roles in the organisation. A manager can not read or write data by default. This role should be appointed separately. 
+
+.. image:: /images/b_usermanagement_02.png
+
+===============	
+User Management
+===============
+
+Users can be managed in the User Management interface.
+This interface can be reached via the {yourorganisation}.lizard.net/management/users/ (or `demo.lizard.net/management/users<https://demo.lizard.net/management/users>`).
+
+.. image:: /images/b_usermanagement_03.png
+
+.. note::
+    You require a “manager” role to access the User Management interface.
+    Haven’t got a “manager” role but you would like to add the User Management interface?
+    Please contact the application manager within your organisation or our support office (servicedesk@nelen-schuurmans.nl)
+	
+Manage Existing users
```

Comment:
```suggestion
Manage existing users
```

Hoofdletters In Titels is erg Amerikaans. Het komt uit de tijd dat typmachines nog niet diverse fonts hadden.

Het moet in ieder geval consistent in het document zijn (=> iets voor een style guide in de README).